.circleci/chain_on_pr.sh

```diff
@@ -1,102 +1,39 @@
-#!/bin/bash -e
+#!/bin/bash
 
-export NVM_DIR="$HOME/.nvm"
-[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
-[ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
+echo "Downloading the current testnet Boyar config.json"
+curl -O $BOOTSTRAP_URL
+echo "Done downloading! Let's begin by cleaning up the testnet of any stale networks for PRs that are already closed"
 
-nvm use $NODE_VERSION
+node .circleci/testnet-cleanup-mark.js
 
-cd .circleci && npm install @orbs-network/orbs-nebula && cd ..
+echo "Copying the newly updated config.json to S3 (with networks to remove)"
+aws s3 cp --acl public-read config.json $BOOTSTRAP_S3_URI
+echo "Done!"
 
-. ./test.common.sh
+sleep 60
+echo "Verifying the networks are being cleaned.."
+node .circleci/testnet-poll-disabled-chains.js
 
-COMMIT_HASH=$(./docker/hash.sh)
+echo "Refreshing config.json and removing the dead networks from it.."
```

Comment:
what's the difference between cleanup and removing dead networks?

Reply:
The mechanics of Boyar at this stage dictates that a you MUST specify in the vchain object the property `"disabled": true`
after which I need to wait to make sure I get a 404 for the metrics path of that vchain to make sure it has been removed. Only then I can completely remove the network from the boyar vchains configuration file.
That's just how it works at the moment. This is old stuff though. Didn't change in this PR

Reply:
@electricmonk  - This is not related to this PR, it is a behavior that was there before this PR.
Removing a vchain requires explicitly setting disabled: true. 
Simply deleting that vchain from the boyar config does not delete it.